src/storage/btree.c

```diff
@@ -22293,10 +22319,13 @@ btree_scan_for_show_index_capacity (THREAD_ENTRY * thread_p, DB_VALUE ** out_val
   db_make_int (out_values[idx], cpc.leaf_pg_cnt);
   idx++;
 
+  db_make_bigint (out_values[idx], cpc.ovfl_oid_pg.tot_pg_cnt);
+  idx++;
+
   db_make_int (out_values[idx], cpc.nleaf_pg_cnt);
   idx++;
 
-  db_make_int (out_values[idx], cpc.tot_pg_cnt);
+  db_make_int (out_values[idx], cpc.tot_pg_cnt + cpc.ovfl_oid_pg.tot_pg_cnt);
```

Comment:
If "cpc.ovfl_oid_pg.tot_pg_cnt" is bigint then "cpc.tot_pg_cnt + cpc.ovfl_oid_pg.tot_pg_cnt" must also be bigint.

Reply:
All variables are int.